ocaml/quicktest/qt.ml

```diff
@@ -5,6 +5,7 @@ type sr_info =
   { sr: API.ref_SR
   ; allowed_operations: API.storage_operations_set
   ; capabilities: string list
+  ; required_sm_api_version : string
```

Comment:
Why not store these as floats or (int * int) for major * minor versions?

Reply:
Because I did not want to parse it, but wanted to keep things simple for now